src/main/java/team/baymax/logic/commands/patient/DeletePatientCommand.java

```diff
@@ -17,7 +17,7 @@
  */
 public class DeletePatientCommand extends Command {
 
-    public static final String COMMAND_WORD = "deletepatient";
+    public static final String COMMAND_WORD = "deletepat";
```

Comment:
shotening to "pat" from "patient" could be confusing for the user as "pat" is not a conventional short form for patient (unlike appt)